src/main/frontend/components/block.cljs

```diff
@@ -2635,14 +2642,15 @@
                       (:block/tags block)
                       (remove (fn [t]
                                 (or (ldb/inline-tag? (:block/raw-title block) t)
-                                    (:logseq.property.class/hide-from-node t)))))
+                                    (:logseq.property.class/hide-from-node t)
+                                    (contains? ldb/internal-tags (:db/ident t))))))
```

Comment:
Like `#Journal`, I think it'd be helpful to display `#Property` and `#Tag` so users can easily see other properties and tags from a page header. I'm happy to enable this if there's no objection

Reply:
I'm fine with this.